packages/mobile/src/components/CodeInput.tsx

```diff
@@ -93,6 +96,27 @@ export default function CodeInput({
                 }
                 onChangeText={onInputChange}
                 multiline={multiline}
+                // This disables keyboard suggestions on iOS, but unfortunately NOT on Android
+                // Though `InputType.TYPE_TEXT_FLAG_NO_SUGGESTIONS` is correctly set on the native input,
+                // most Android keyboards ignore it :/
```

Comment:
It's unfortunate that we need this because of the Android fragmentation, but great job finding a solution 😄